src/controllers/users.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+const usersService = require('../services/users');
+const { ApiError } = require('../exceptions/ApiError');
+
+const getAll = async(req, res) => {
+  const users = await usersService.getAll();
+
+  res.send(users);
+};
+
+const getById = async(req, res) => {
+  const { id } = req.params;
+  const user = await usersService.getById(id);
+
+  if (!user) {
+    throw ApiError.NotFound();
```

Comment:
Use `new` keyword when throwing ApiError.NotFound() to create a new instance of the error.